README.md

```diff
@@ -10,6 +10,42 @@ If there is anything you are looking for that is not covered here you should ref
 
 ----
 
+### File structure ###
```

Comment:
I would agree with Chris. I'd say it would make more sense into a best/preferred practices. For the obj-C guide we kept the extra rules that apply only to use into a style guide extension.. That would make sense here as well. But we can discuss tomorrow